formats/shared/definitions/routes_redirects.jsonnet

```diff
@@ -62,17 +62,7 @@
           "exact",
         ],
       },
-      destination: {
-        type: "string",
-        anyOf: [
-          {
-            "$ref": "#/definitions/absolute_fullpath",
-          },
-          {
-            "$ref": "#/definitions/govuk_subdomain_url",
-          },
-        ],
-      },
+      destination: { type: "string", format: "uri" },
```

Comment:
So we still have a little validation to make sure we're not just adding a generic string. Nice!

I was a bit concerned that perhaps relative paths wouldn't be allowed, but it looks like they are: https://datatracker.ietf.org/doc/html/rfc3986